mne/viz/topomap.py

```diff
@@ -757,19 +758,53 @@ def plot_topomap(data, pos, vmin=None, vmax=None, cmap=None, sensors=True,
     %(topomap_border)s
     %(topomap_ch_type)s
 
+        ..versionadded:: 0.24.0
+    cnorm : matplotlib.colors.Normalize | None
+        Colormap normalization, default None means linear normalization. If not
+        None, ``vmin`` and ``vmax`` arguments are ignored. See Notes for more
+        details.
+
     Returns
     -------
     im : matplotlib.image.AxesImage
         The interpolated data.
     cn : matplotlib.contour.ContourSet
         The fieldlines.
+
+    Notes
+    -----
+    The ``cnorm`` parameter can be used to implement custom colormap
+    normalization. By default, a linear mapping from vmin to vmax is used,
+    which correspond to the first and last colors in the colormap. This might
+    be undesired when vmin and vmax are not symmetrical around zero (or a value
+    that can be interpreted as some midpoint). For example, assume we want to
+    use the RdBu colormap (red to white to blue) for values ranging from -1 to
+    3, and 0 should be white. However, white corresponds to the midpoint in the
+    data by default, i.e. 1. Therefore, we use the following colormap
+    normalization ``cnorm`` and pass it as the the ``cnorm`` argument:
+
+        from matplotlib.colors import TwoSlopeNorm
+        cnorm = TwoSlopeNorm(vmin=-1, vcenter=0, vmax=3)
+
+    Note that because we already defined ``vmin`` and ``vmax`` in the
+    normalization, arguments ``vmin`` and ``vmax`` are ignored if these are
+    also passed.
+    See https://matplotlib.org/stable/tutorials/colors/colormapnorms.html for
```

Comment:
there's probably a way to link to this using an intersphinx crossref, but my attempt to get it to work locally failed, so just using the URL is fine I guess.  (the advantage of a proper crossref is that our doc build would notice if it failed to resolve someday, whereas a URL-as-text is not tested)

Reply:
In that case I'd also prefer a proper crossref. Where would I need to add it? Locally at the end of the docstring?

Reply:
```
$ python -m sphinx.ext.intersphinx https://matplotlib.org/objects.inv > matplotlib.txt
$ cat matplotlib.txt | grep colormapnorms
	tutorials/colors/colormapnorms           Colormap Normalization                  : tutorials/colors/colormapnorms.html
	sphx_glr_download_tutorials_colors_colormapnorms.py tutorials/colors/colormapnorms.html#sphx-glr-download-tutorials-colors-colormapnorms-py
	sphx_glr_tutorials_colors_colormapnorms.py Colormap Normalization                  : tutorials/colors/colormapnorms.html#sphx-glr-tutorials-colors-colormapnorms-py
```
Then if you look at the `.txt` above this line you see it's in `std:doc` which means use can use `:doc:` to get to it. Pushed a commit for this:

![Screenshot from 2021-06-23 11-57-58](https://user-images.githubusercontent.com/2365790/123129870-46620800-d41a-11eb-8cf9-024eb5a5d936.png)